src/Integration/Binding/BindingController.cs

```diff
@@ -116,23 +116,28 @@ private void OnBind(BindCommandArgs args)
 
         void IBindingWorkflowExecutor.BindProject(BindCommandArgs bindingArgs)
         {
-            var workflow = CreateBindingWorkflow(bindingArgs);
+            var workflow = CreateBindingWorkflow(host, bindingArgs);
 
             IProgressEvents progressEvents = workflow.Run();
             Debug.Assert(progressEvents != null, "BindingWorkflow.Run returned null");
             this.SetBindingInProgress(progressEvents, bindingArgs);
         }
 
-        internal /* for testing purposes */ IBindingWorkflow CreateBindingWorkflow(BindCommandArgs bindingArgs)
+        internal static /* for testing purposes */ IBindingWorkflow CreateBindingWorkflow(IHost host, BindCommandArgs bindingArgs)
```

Comment:
Making the method static makes writing unit tests easier - it avoids having to configure the set of services required by the constructor.